tests/operators/test_redshift_to_s3_operator.py

```diff
@@ -51,23 +52,25 @@ def test_execute(self, mock_run, mock_session):
             redshift_conn_id="redshift_conn_id",
             aws_conn_id="aws_conn_id",
             task_id="task_id",
+            table_as_file_name=table_as_file_name,
             dag=None
         ).execute(None)
 
         unload_options = '\n\t\t\t'.join(unload_options)
+        s3_key = '{}/{}_'.format(s3_key, table) if table_as_file_name else s3_key
```

Comment:
This isn't needed in the test - we call it explicitly with one value so we should have this as
```suggestion
        s3_key = '{}/{}_'.format(s3_key, table)
```

Also you need to add a test covering `table_as_file_name=False`

Reply:
Infact - with the table_as_file_name=True the test code shouldn't need changing _at all_ - please remove these changes to prove that the existing behaviour hasn't changed :)

Reply:
Thank you very much ashb for your help. I have removed the changes and added a case for table_as_file_name = False. However, I am not sure that I have understood you completely